src/main/webapp/src/pages/HelpPage.js

```diff
@@ -1,60 +1,48 @@
-import React, { Component } from 'react';
-import 'bulma/css/bulma.css';
+import React, { useState, useEffect } from 'react';
 import '../css/home.css';
 
 // Import Bootstrap CSS and JS. Only this page uses these. 
 import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap/dist/js/bootstrap.js';
 
+// This is a stateless, functional React component. 
+const ResourceCard = ({ organization, description, url }) => (
+  <div className = "p-3">
+    <div className="card">
+      <div className="card-body">
+        <h5 className="card-title text-info">{organization}</h5>
+        <p className="card-text text-secondary" >{description}</p>
+        <a className="card-text btn btn-light" href={url}>View more</a>
+      </div>
+    </div>
+  </div>
+);
+
 /* Component for home page */
-class HelpPage extends Component {
+// class HelpPage extends Component {
+const HelpPage = () => {
 
-  constructor(props) {
-    super(props);
-    this.state = {
-      resources: [],
-    };
-  }
+  const [resources, updateResources] = React.useState([]);
 
-  /* This is a lifecycle method of the class component. The code inside it is run whenever a component is ready to be loaded onto the page. */
-  componentDidMount() {
+  useEffect(() => {
```

Comment:
the useEffect hook is called after the component is rendered. Since the call to this api is the content that is to be rendered, I think it would be better to just call it in the component body itself before render. Since whenever state changes, the component re-renders.